lib/ts/recipe/dashboard/api/userroles/addRoleToUser.ts

```diff
@@ -0,0 +1,52 @@
+import { APIInterface, APIOptions } from "../../types";
+import UserRoles from "../../../userroles";
+
+import STError from "../../../../error";
+
+const addRoleToUser = async (
+    _: APIInterface,
+    tenantId: string,
+    options: APIOptions,
+    __: any
+): Promise<{
+    status: "OK" | "UNKNOWN_ROLE_ERROR" | "ROLE_ALREADY_ASSIGNED";
+}> => {
+    const requestBody = await options.req.getJSONBody();
+
+    const userId = requestBody.userId;
+    const role = requestBody.role;
```

Comment:
in all these APIs, how is capital letters handled in case of roles and permissions? Does the core lower case them? If yes, will that be properly reflected on the frontend?